grpc-front/src/service/function.rs

```
@@ -71,9 +71,17 @@ impl<T: FunctionGrpc + Tracing + Send + Debug + Sync + 'static> FunctionService
                                     Ok(Some(runner)) => {
                                         // Check if the worker is associated with the runner
                                         if runner.id == Some(rid) {
-                                            functions.push(convert_worker_to_function_specs(
+                                            // warn only
+                                            let _ =  convert_worker_to_function_specs(
                                                 wid, data, runner,
-                                            ));
+                                            ).map(|r| {
+                                                functions.push(r);
+                                            }).inspect_err(|e|
```

ignore error (warn only)